envoy-control-core/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/groups/GroupChangeWatcher.kt

```diff
@@ -41,12 +41,13 @@ internal class GroupChangeWatcher(
 
     override fun createWatch(
         ads: Boolean,
-        request: DiscoveryRequest,
-        knownResourceNames: MutableSet<String>,
-        responseConsumer: Consumer<Response>,
+        request: XdsRequest?,
+        knownResourceNames: MutableSet<String>?,
```

Comment:
Are `?` necessary?

Reply:
Good catch, should probably be reverted, probably intellij done this without my knowledge